add new wordle plugin

cibere opened the pull request.

jjw24 reviewed (commented):
First ever game you can play with flow 😀

Your release didn't generate the zip file, check your requirements I couldn't browse to that commit.

cibere commented:
> First ever game you can play with flow 😀
> 
> Your release didn't generate the zip file, check your requirements I couldn't browse to that commit.

Sorry about the release, I changed my build workflow and forgot to test it. I've reverted it back to the one I use with my other plugins and updated the download link.

jjw24 reviewed (approved):
👍
